### ISSUE TITLE
Catalog order change in RTD?

### ISSUE THREAD

**Issue by andrewkern** [popsim-consortium/stdpopsim#531]
The tab organization of our critters in the catalog on RTD doesn't make any sense. Should we group by clade? Should we alphabetize? 
![image](https://user-images.githubusercontent.com/7674197/81875546-58922700-9535-11ea-9832-4072f84c88c3.png)

This comes up again in the manuscript, where the ordering of species names mimics what is seen on RTD

**Comment by petrelharp** [comment]
It should be in order from "worst" to "best".

**Comment by andrewkern** [comment]
i second the motion to put the Brassica in the "best" slot

**Comment by jeromekelleher** [comment]
Alphabetical seems like the least arbitrary choice here. It's an easy fix: https://github.com/popsim-consortium/stdpopsim/blob/master/docs/catalog.rst

**Comment by andrewkern** [comment]
bah would be an easy fix if it didn't throw weird errors when the order is switched.... getting a `(ERROR/3) Document may not end with a transition` error....

**Comment by jeromekelleher** [comment]
Are you leaving space etc between the groups like there was before? Rst is finicky.

**Comment by andrewkern** [comment]
yup positive... it's something about HomSap or PonAbe being the last entry... not sure what is different and very hard to get a error message out...

**Comment by andrewkern** [comment]
ugh so something is going wrong for those two entries if they are last in the `catalog.rst` file. @petrelharp found a hack to fix things by adding another heading block past the last species such that
```
.. speciescatalog:: EscCol

.. speciescatalog:: PonAbe

=====
foo
=====
```
Perhaps a value is missing from HomSap and PonAbe that is throwing things off in rst generation?

**Comment by jeromekelleher** [comment]
Closed in #532
